client/src/main/scala/drt/client/components/MonthlyStaffing.scala

```diff
@@ -104,13 +123,9 @@ object MonthlyStaffing {
       }.toTagMod)
   }
 
-  def consecutiveDaysInMonth(startDay: SDateLike, endDay: SDateLike): Seq[SDateLike] = {
-    val days = (endDay.getDate - startDay.getDate) + 1
-    List.tabulate(days)(i => startDay.addDays(i))
-  }
 
   def sixMonthsFromFirstOfMonth(date: SDateLike): Seq[SDateLike] = (0 to 5)
-    .map(i => SDate.firstDayOfMonth(date).addMonths(i))
+    .map(i => if (i == 0) SDate.now() else SDate.firstDayOfMonth(date).addMonths(i))
```

Comment:
Making the first date `now()` breaks what you would expect this function to be doing. If this hack is needed then it doesn't belong here, but I'm still not clear why it's needed